tests/apollo/test_skvbc_view_change.py

```diff
@@ -155,8 +155,7 @@ class SkvbcViewChangeTest(unittest.TestCase):
 
             await tracker.tracked_read_your_writes()
 
-            async with trio.open_nursery() as nursery:
-                nursery.start_soon(tracker.run_concurrent_ops, 100)
+            tracker.run_concurrent_ops(100)
```

Comment:
Please, be careful when invoking coroutines. This should say:
`await tracker.run_concurrent_ops(100)`